File: Python/python_prac/poly.cpp
#include<iostream>
#include<string>
using namespace std;
int main()
{

class Car
{
    public:
    Car()
    {
        cout<<" I am in constructor\n";
    }
    virtual void start()=0;


    virtual void stop()=0;
};

class Innova:public Car
{
    public:
    Innova()
    {
        cout<<" I am inside Innova class\n";
    }
    void start()
    {
        cout<<" Innova Started\n";
    }

    void stop()
    {
        cout<<" Innova stop\n";
    }
};

class Swift:public Car
{
    public:
    Swift()
    {
        cout<<" I am inside Swift class\n";
    }

    void start()
    {
        cout<<" Swift Started\n";
    }

    void stop()
    {
        cout<<" Swift stop\n";
    }
};

Car *object = new Innova();
object->start();
}

File: Programs/Python_developer/class/poly.py
class Car():
    def __init__(self,model):
        self.model=model
        print("I am under parent Car Class")

class Innova(Car):
    def __init__(self,status,model):
        super().__init__(model)
        self.status=status
                
    def start(self):
        print(f"Innova Car started with {self.status} status with model {self.model}")
        
class Swift(Car):
    def __init__(self,status):
        self.status=status
                
    def start(self):
        print(f"Swift Car started with {self.status}")
        
        
Innova1=Innova('on',2019)
Innova1.start()
Innova1=Swift('on')
Innova1.start()
